Challenge1.js

```diff
@@ -0,0 +1,35 @@
+const studentData = [{
+    name: 'Bobby',
+    scores: [62,88,91,56,70]
+},{
+    name: 'Katie',
+    scores: [93,70,66,85,89]
+},{
+    name: 'Michael',
+    scores: [61,82,90,79,89]
+},{
+    name: 'Sarah',
+    scores: [80,94,86,91,81]
+}]
+
+const convertData = (studentData) => {
```

Comment:
This is good.  Could you simplify this and make it shorter, as short as possible?